Islandora-CLAW/islandora_demo
# Issue 1045

**GitHub Issue**: Resolves https://github.com/Islandora-CLAW/CLAW/issues/1045

# What does this Pull Request do?

Adds the "Parent node has term" condition to the Audio Original File context.  This way, both the media must be an Original File and the parent node must be tagged as an Audio model in order to trigger derivatives.

# What's new?
Some changed yml.

# How should this be tested?

Add dsms either in hooks for derivatives or in the EmitEvent action to see when things get triggered.  Before the PR, an Orginal File for an Image would trigger audio derivatives in addition to image derivatives.  After the PR, it should only trigger image derivatives.  Of course, making an Original File that's owned by an Audio model object should trigger audio derivatives only, too.

# Interested parties
@kayakr @Islandora-CLAW/committers
